.github/workflows/update_api_database.yml

```diff
@@ -96,10 +123,14 @@ jobs:
 
       - name: Install dependencies
         run: |
-          brew install postgresql@15
           brew install minio/stable/mc
           mc alias set spaces $DO_SPACES_ENDPOINT $DO_SPACES_ACCESS_KEY_ID $DO_SPACES_SECRET_ACCESS_KEY
           python3 -m pip install --force-reinstall -r requirements.txt
+
+          postgres --version
```

Comment:
Is this stdout just for logging or debugging purposes?

Reply:
yup! postgres version mismatch was the biggest issue I hit so this seemed worth keeping along with the other dependency logging after it